bootstraps/stage1/spack.in.yaml

```diff
@@ -0,0 +1,28 @@
+# This is a Spack Environment file.
+#
+# It describes a set of packages to be installed, along with
+# configuration settings.
+spack:
+  specs:
+  - binutils +stage0
+  - gcc +stage0 +binutils ~bootstrap languages=c,c++
+  - gawk
+  - bison
+  view: true
+  concretizer:
+    unify: true
+
+  compilers:
```

Comment:
Preferably `spack -e stage1 compiler find`

Reply:
Now that we’re building up a driving script I think I agree. This was the solution to making as much of it as possible defined in the environment. Either way it needs to be in the env, because we need to control which gcc@13 is selected, and it’s the only way I’ve found.